tasks/__init__.py

```diff
@@ -504,6 +519,10 @@ def test_travis_else():
     karma(single=True, browsers='PhantomJS')
 
 @task
+def test_travis_varnish():
```

Comment:
`test_travis_osf` and `test_travis_else` only exist to make it easier to split Travis runs into multiple, parallel builds. `test_travis_varnish` doesn't really contribute to that, so it can probably be removed.